PySDM/physics/constants_defaults.py

```diff
@@ -342,3 +342,13 @@ def compute_derived_values(c: dict):
     c["water_molar_volume"] = c["Mv"] / c["rho_w"]
     c["rho_STP"] = c["p_STP"] / c["Rd"] / c["T_STP"]
     c["H_u"] = c["M"] / c["p_STP"]
+
+
+B80W_G0 = -2.9912729e3 * si.hPa
```

Comment:
in the paper, g0 is multiplied by T^{-2} to give something nondimensional (as it is under the exponent), so g0 has a unit of K^2, g1 has a unit of K, g0 is non-dimensional, ...

Reply:
g0 has a unit of K^2